external-crates/move/crates/move-compiler/src/expansion/path_expander.rs

```diff
@@ -469,6 +473,12 @@ impl Move2024PathExpander {
                     }
                 }
 
+                if incomplete {
+                    result = NR::ResolutionFailure(
+                        Box::new(result),
+                        NF::InvalidKind("fully defined access chain".to_string()),
```

Comment:
Instead of this, I think you should add a new failure kind of `IncompletePath` that holds the _whole_ path location, and the error should look like this:

```
error[E03006]: unexpected name in this position
   ┌─ tests/move_2024/ide_mode/colon_colon_incomplete.move:14:9
   │
14 │         a::
   │         ^^^ Incomplete name in this position. Expected an identifier after '::'
```

Reply:
Done!